_examples/ThisExtensionFunc.md

```diff
@@ -0,0 +1,62 @@
+---
+title: this in extension functions
+---
+    
+<div class="sample" markdown="1">
+
+Normally, when you create a class, by default it inherits from *Any*, the Kotlin default supertype which declares toString(), hashCode() and equals().
+In other languages like Java, if you want to extend a class behaviour, you create a *derived* class like this:
```

Comment:
behavior 
if we are keeping Ktor as American English?

Reply:
you would need to create a *derived* class like this:

Reply:
:%s/behaviour/behavior/g 

I learned my lesson! :D